src/providers/UserProvider.tsx

```diff
@@ -40,7 +43,10 @@ export default function UserProvider({ children, data }: UserProviderProps): JSX
                   strings.UPDATED_TIMEZONE_MSG,
                   <Link to={APP_PATHS.MY_ACCOUNT}>{strings.MY_ACCOUNT}</Link>
                 ),
-                strings.formatString(strings.UPDATED_TIMEZONE_TITLE, foundTimeZone ? browserTimeZone : strings.UTC)
+                strings.formatString(
+                  strings.UPDATED_TIMEZONE_TITLE,
+                  foundTimeZone.id === 'Etc/UTC' ? foundTimeZone.longName : foundTimeZone.id
```

Comment:
this should be foundTimeZone.longName for all zones (this is the display name for the time zone id)